src/Catalyst.Node.POA.CE.Tests/UnitTests/Config/NetworkConfigTests.cs

```diff
@@ -0,0 +1,89 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using Autofac;
+using Autofac.Configuration;
+using Catalyst.Core.Lib.Config;
+using Catalyst.Core.Lib.P2P;
+using Catalyst.Protocol.Network;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using SharpRepository.Repository;
+using Xunit;
+
+namespace Catalyst.Node.POA.CE.Tests.UnitTests.Config
+{
+    public sealed class NetworkConfigTests
+    {
+        public static readonly List<object[]> NetworkFiles;
+
+        static NetworkConfigTests()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Initialize all 'static fields' inline and remove the 'static constructor'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)